src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -264,10 +264,8 @@ public void onCommandExecuted(CommandExecuted commandExecuted)
 							handleHydratePrevCommand();
 							break;
 						case RESET:
-							handleHydrateResetCommand();
-							break;
 						case HYDRATE:
-							commandDelegate.invokeCommand(commandExecuted);
+							commandDelegate.invokeCommand(commandExecuted); // TODO remove this and use the generic invoke after all commands are refactored
```

Comment:
@harikesh409 It looks like this logic was added back in during one of your merge conflict resolutions.

Can you make sure that there are no diffs for other commands that you did not change?

Reply:
I have just merged the `REST` and `HYDRATE` cases as IntelliJ was giving me a warning.

Reply:
Ah nevermind, I misunderstood this change. Everything looks good so I'll go ahead and merge now :)

Good work!